client-linuxapp/src/serviceinfo.rs

```diff
@@ -74,22 +74,37 @@ fn create_user(user: &str) -> Result<()> {
     Ok(())
 }
 
+fn check_password_encryption(s: &str) -> bool {
+    let prefixes = ["$2b$", "$6$", "$5$"];
+
+    for prefix in prefixes.iter() {
+        if s.starts_with(prefix) {
+            return true;
+        }
+    }
+
+    return false;
```

Comment:
no need for the `return false;` there, simply `false` and the expression will be returned